.slashrc

```diff
@@ -341,6 +344,36 @@ class MediaPlugin(slash.plugins.PluginInterface):
       slash.context.session.results.global_result.get_log_dir(), "results.xml")
     tree.write(filename)
 
+  def _load_blacklist(self, file_path = None):
+    if file_path is None or len(file_path) == 0 or not os.path.exists(file_path):
+      return
+
+    import json, glob
+    for file in glob.glob(os.path.join(str(file_path), "*.json")):
+      with open(file, 'rb') as f:
+        self.blacklist.update(json.load(f))
+
+  def _assert_not_matches(self, test, filter, message):
+    from slash.utils.pattern_matching import Matcher
+    search= [
+      str(test.__slash__),
+      "driver=" + self._get_driver_name(),
+      "platform=" + self._get_platform_name(),
+    ]
+    assert not Matcher(filter).matches(" ".join(search)), message
+
+  def _assert_not_matches(self, test):
+    # Current platform and driver combine not in blacklist
+    from slash.utils.pattern_matching import Matcher
+    search= [
+      str(test.__slash__),
+      "driver=" + self._get_driver_name(),
+      "platform=" + self._get_platform_name(),
+    ]
+
+    for filter,message in self.blacklist.items():
+      assert not Matcher(filter).matches(" ".join(search)), message
+
```

Comment:
Two functions with the same name in the same namespace?  This is not appropriate in python.  The first definition gets shadowed (replaced) by the python interpreter with the second definition.  Please delete the first definition (it's not used anywhere).   And please appropriately rename the second definition to `_assert_not_blacklisted`.